platform/android/display_server_android.cpp

```diff
@@ -138,6 +138,20 @@ Size2i DisplayServerAndroid::screen_get_size(int p_screen) const {
 	return OS_Android::get_singleton()->get_display_size();
 }
 
+float DisplayServerAndroid::screen_get_refresh_rate(int p_screen) const {
+	print_line("Test for getting Display.Mode from JNI");
+	JNIEnv *env = get_jni_env();
+	jclass display_class = env->FindClass("android/view/Display");
+	jmethodID get_mode_id = env->GetMethodID(display_class, "getMode", "()Landroid/view/Display/Mode");
+	if(get_mode_id == 0) {
+		return 60.0;
+	}
+	else {
+		print_line("Got method id");
+		return -1.0;
+	}
```

Comment:
I would go through the `GodotIOJavaWrapper` to match the pattern in the other methods.

Reply:
Done in the most recent commit